src/shared/ui/link-custom/link-custom.tsx

```diff
@@ -28,14 +30,22 @@ const linkCustomVariants = cva('', {
   defaultVariants: { variant: 'textLink' },
 });
 
+const externalLinkAttributes = {
+  target: '_blank',
+  rel: 'noopener noreferrer',
+};
+
 export const LinkCustom = ({
   children,
   href,
   icon = <></>,
   className = '',
-  variant,
+  variant = 'textLink',
+  external = false,
   ...props
 }: LinkCustomProps) => {
+  const iconElement = external && variant === 'textLink' ? <TextLinkIcon /> : icon;
```

Comment:
POV: We should either change logic and put all icons here and take them based on variants, or should leave it as previous and provide icons through parents. Why we should make exception for TextLinkIcon. The behavior of component became inconsistent.

Reply:
Actually i think same, as far as i see all variants except textLink has an arrow icon, so why should we pass it manually if can show it by default. But lets see what others think about it.

Reply:
i agree that the better option would be to make consistent icons that related to each variant, this will:
1. reduce complexity because component will know exactly what icon should it render by itself based on the variant
2. if we some day want to change the icon on all links we will need to change it only in one place

Reply:
done [37b266c](https://github.com/rolling-scopes/site/pull/467/commits/37b266c31b196f322fd6f3880da0bd6135e30a7e)